admin/sql/updates/2019-02-13-change-login-id-type.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+ALTER TABLE "user" DROP COLUMN user_login_id;
+ALTER TABLE "user" ADD COLUMN login_id TEXT NOT NULL DEFAULT uuid_generate_v4()::text;
+
+UPDATE "user"
+   SET login_id = id::text;
+
+ALTER TABLE "user" user_login_id_key UNIQUE (login_id);
+CREATE UNIQUE INDEX login_id_ndx_user ON "user" (login_id);
```

Comment:
I believe that as an implementation detail, in postgres unique constraints are implemented by creating a unique index. With a constraint _and_ index on this field, does it mean that there are two indexes? Do we mind?

Reply:
this is how it is done for other columns, i'd keep it as is and change them all if needed.

Reply:
Fine